app.js

```diff
@@ -11,3 +11,47 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   navMenu.classList.remove('active');
   document.querySelector('body').classList.toggle('active');
 }));
+// Get Data
+const form = document.querySelector('form');
+const nameInput = document.querySelector('.name');
+const email = document.querySelector('.email');
+const message = document.querySelector('#message');
+const success = document.querySelector('#success');
+const errorNodes = document.querySelectorAll('.error');
+form.addEventListener('submit', (ev) => {
+  function validateForm() {
+    // clear error / success messages
+    function clearMessage() {
+      for (let i = 0; i < errorNodes.length; i += 1) {
+        errorNodes[i].innerText = '';
+      }
+      success.innerText = '';
+    }
+    clearMessage();
+    let errorFlag = false;
+    if (nameInput.value.length < 1) {
+      ev.preventDefault();
+      errorNodes[0].innerText = 'Name can not be blank';
+      errorFlag = true;
+    }
+    // check if email is valid
+    function emailIsValid(email) {
+      const pattern = /\S+@\S+\.\S+/;
+      return pattern.test(email);
+    }
+    if (!emailIsValid(email.value)) {
+      ev.preventDefault();
+      errorNodes[1].innerText = 'Invalid Email Address';
+      errorFlag = true;
+    }
```

Comment:
- [ ] Hi @Team, no error is shown when I submit the contact form with the email field containing uppercase characters. Please let's fix this and get closer to approval. 
![Screenshot from 2022-03-12 02-32-23](https://user-images.githubusercontent.com/63087491/157998628-16fa541d-66b2-4f43-8481-43a5fbb73416.png)
.